src/storage/useFilePreview.ts

```diff
@@ -0,0 +1,46 @@
+'use client'
+
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { useMemo } from 'react'
+import { useAppwrite } from 'react-appwrite'
+import { Image } from './types'
+
+/**
+ * Get a file preview of an image.
```

Comment:
it doesn't return preview of an image
should be `Get a file preview image`

Reply:
Yep, I've updated both. Good catch :)